assets/src/blocks/HubspotForm/HubspotFormThankyou/HubspotFormThankYouBlock.js

```diff
@@ -0,0 +1,50 @@
+import { registerBlockType } from '@wordpress/blocks';
+import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+
+const { __ } = wp.i18n;
+
+const BLOCK_NAME = 'planet4-blocks/hubspot-form-thankyou';
+
+export const registerBlock = () => {
+  return registerBlockType(BLOCK_NAME, {
+    title: 'Hubspot Form Thank You Message (beta)',
```

Comment:
I suppose we can generalize this block to `Form Thank You Message`? The logic inside the block is not specific to HS.

Reply:
That would also shorten quite a few directory and file names.